examples/stack_analysis/stack_analysis.go

```diff
@@ -54,7 +54,7 @@ var (
 
 func init() {
 	questions := f.TextFile(
-		"Posts100k.xml", 4,
+		"/Posts100k.xml", 4, // needs to be a full qualified path to run on an agent
```

Comment:
With this change you wouldn't be able to run the example in the current folder. On the other hand the relative path won't work if you'd run it distributed. The agents will need the same relative path setup for the executable binary. Therefore this code makes assumptions how agents are configured. I'll make it works in both scenarios.